packages/core/src/overlay/OverlayPanel.css

```diff
@@ -18,3 +18,8 @@
   z-index: var(--saltOverlay-zIndex, var(--salt-zIndex-flyover));
   width: max-content;
 }
+
+.saltOverlayPanel-inner {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
Is this solution based on the fact that `OverlayTrigger` in examples is added before content?

Reply:
Ah, not OverlayTrigger, but OverlayCloseButton... For some reason I thought it was built into OverlayPanel (so always in the same source order). Might switch back to absolute positioning now...

Reply:
Reverted back to the absolutely positioned option, which puts `OverlayCloseButton` in the top-right regardless of source order but still keeps `content | scrollbar | closebutton` alignment.